h1st/h1st/core/ensemble.py

```diff
@@ -2,106 +2,71 @@
 import pandas as pd
 from sklearn.multioutput import MultiOutputClassifier
 from sklearn.ensemble import RandomForestClassifier
-from sklearn.linear_model import LogisticRegression, LogisticRegressionCV
+from sklearn.preprocessing import RobustScaler
+from sklearn.metrics import confusion_matrix
+from sklearn.metrics import precision_recall_fscore_support
+from sklearn.metrics import accuracy_score
 
 from h1st.core.model import Model
 
 
 class StackEnsemble(Model):
-    def __init__(self, models, ensembler):
-        self.models = models
-        # must have .fit() / .predict(), currently assumes sklearn
-        self.model = ensembler
+    def __init__(self, ensembler, sub_models):
```

Comment:
pls consider adding typing for parameters. Ex: def __init__(self, ensembler: Model, sub_models: List[Model])